includes/admin/discounts/edit-discount.php

```diff
@@ -127,186 +128,162 @@
+									<option value="any"<?php selected( 'any', $condition ); ?>><?php printf( __( 'Cart needs one or more of the selected %s', 'easy-digital-downloads' ), edd_get_label_plural() ); ?></option>
+								</select>
+							</p>
+							<p>
+								<label>
+									<input type="radio" class="tog" name="scope" value="global"<?php checked( 'global', $discount->scope ); ?>/>
+									<?php _e( 'Apply discount to entire purchase.', 'easy-digital-downloads' ); ?>
+								</label><br/>
+								<label>
+									<input type="radio" class="tog" name="scope" value="not_global"<?php checked( 'not_global', $discount->scope ); ?>/>
+									<?php printf( __( 'Apply discount only to selected %s.', 'easy-digital-downloads' ), edd_get_label_plural() ); ?>
+								</label>
+							</p>
+						</div>
+						<p class="description"><?php printf( __( '%s this discount can only be applied to. Leave blank for any.', 'easy-digital-downloads' ), edd_get_label_plural() ); ?></p>
+					</td>
+				</tr>
+
+				<?php do_action( 'edd_edit_discount_form_before_excluded_products', $discount->id, $discount ); ?>
+
+				<tr>
+					<th scope="row" valign="top">
+						<label for="edd-excluded-products"><?php printf( __( 'Excluded %s', 'easy-digital-downloads' ), edd_get_label_plural() ); ?></label>
+					</th>
+					<td>
+						<?php echo EDD()->html->product_dropdown( array(
+							'name'        => 'excluded_products[]',
+							'id'          => 'excluded_products',
+							'selected'    => $excluded_products,
+							'multiple'    => true,
+							'chosen'      => true,
+							'variations'  => true,
+							'placeholder' => sprintf( __( 'Select %s', 'easy-digital-downloads' ), edd_get_label_plural() )
+						) ); ?>
+						<p class="description"><?php printf( __( '%s this discount cannot be applied to. Leave blank for none.', 'easy-digital-downloads' ), edd_get_label_plural() ); ?></p>
+					</td>
+				</tr>
+
+				<?php do_action( 'edd_edit_discount_form_before_start', $discount->id, $discount ); ?>
+
+				<tr>
+					<th scope="row" valign="top">
+						<label for="edd-start"><?php _e( 'Start date', 'easy-digital-downloads' ); ?></label>
+					</th>
+					<td>
+						<input name="start_date" id="edd-start" type="text" value="<?php echo esc_attr( false !== $discount->start_date ? $start_date : '' ); ?>" class="edd_datepicker" data-format="<?php echo esc_attr( edd_get_date_picker_format() ); ?>" placeholder="<?php echo esc_attr( edd_get_date_picker_format() ); ?>" />
+
+						<label class="screen-reader-text" for="start-date-hour">
+							<?php esc_html_e( 'Start Date Hour', 'easy-digital-downloads' ); ?>
+						</label>
+						<input type="number" min="0" max="24" step="1" name="start_date_hour" id="start-date-hour" value="<?php echo esc_attr( false !== $discount->start_date ? $start_hour : '0' ); ?>" />
```

Comment:
```suggestion
						<input type="number" min="0" max="24" step="1" name="start_date_hour" id="start-date-hour" value="<?php echo esc_attr( false !== $discount->start_date ? $start_hour : '00' ); ?>" />
```

For clarity on date formatting and to be consistent with the default on Add Discount page.

Alternatively, it could default to blank. It is a bit weird to have the date field be blank but have the times filled out.

![Screenshot from 2020-05-27 15-53-59](https://user-images.githubusercontent.com/6324272/83036276-500c1700-a032-11ea-81d6-8a7107948d33.png)

I'm fine with either though.

On a related note, should we add placeholders for clarity? like `placeholder="hh"` and `placeholder="mm"` ?

Reply:
@ashleyfae I like the `00` because it helps make it clear it's 24 hour format. The `hh` and `mm` placeholders might be confusing for those who do not understand the formatting notation. We could do `00` placeholders as well.

I know it's not consistent with the current `yyyy-mm-dd` placeholder for the date, but I don't think that's great to have there either -- but a discussion for a different PR.

Reply:
Or perhaps they should _actual_ blank values instead of `00` to better match the "leave blank" descriptions below.

Reply:
How about blank values with `00` placeholders?

Also we could adjust the description to read _"Pick the date **and time** .."_ (bold = new)